src/digraph.rs

```diff
@@ -2288,34 +2288,81 @@ impl PyDiGraph {
     ///
     /// This will create a new :class:`~retworkx.PyGraph` object from this
     /// graph. All edges in this graph will be created as undirected edges in
-    /// the new graph object.
+    /// the new graph object. For directed graphs with bidirectional edges, you
+    /// can set `multigraph=False` to condense them into a single edge and specify
+    /// a function to combine the weights/data of the edges.
     /// Do note that the node and edge weights/data payloads will be passed
     /// by reference to the new :class:`~retworkx.PyGraph` object.
     ///
+    /// :param bool multigraph: If set to `False` the output graph will not
+    ///     allow parallel edges. Default: `True`.
```

Comment:
We should explain a bit more here about what it does when it encounters a parallel edge (both with and without `weight_combo_fn` set). When I read this my first thought would be that if `multigraph=False` it raises when there are parallel edges because they're not allowed

Reply:
```suggestion
    ///     allow parallel edges. Instead parallel edges will be condensed
    ///     into a single edge and their data will be combined using
    ///     `weight_combo_fn`. If `weight_combo_fn` is not provided, the data
    ///     of the edge with the largest index will be kept. Default: `True`.
```